test/e2e/README.md

```diff
@@ -221,4 +258,4 @@ All flags of performance tests includes:
 * `performance.http.concurrency (int)`: Number of allowed concurrent http requests (default 1)
 * `performance.http.requests (int)`: Total Number of http requests
 * `performance.http.policy_rules (int)`: Number of CIDRs in the network policy
-* `performance.realize.timeout (duration)`: Timeout of the realization of network policies (default 5m0s)
+* `performance.realize.timeout (duration)`: Timeout of the realization of network policies (default 5m0s)
```

Comment:
You removed the EOF this time. Can you keep it unchanged? The original EOF was correct. In https://github.com/antrea-io/antrea/pull/3045#discussion_r769430101, I was only asking to remove the extra ending line as there were 2.